src/components/nodes/intermediateMessageCatchEvent/webHookFormGroup.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="web-hook">
+    <label for="webHook">Web Hook</label>
+    <input
```

Comment:
Did you try using `FormInput` from `@processmaker/vue-form-elements` here? Take a look at how `CallActivityFormSelect.vue` is set up.

Reply:
copyToClipboard() method does not work with `FormInput` because `.select();` does not exists on  `FormInput`